x-pack/plugins/maps/index.js

```diff
@@ -94,14 +108,14 @@ export function maps(kibana) {
               all: ['map'],
               read: ['config', 'index-pattern']
             },
-            ui: ['save'],
+            ui: ['save', 'show'],
```

Comment:
I don't know much about how the feature registry works, so bear with me. Why did this need to change in this PR? (Canvas as well).

Reply:
This is post rbac feature controls merge. In order to be able to see a map in the app, you need the standard `show` privilege and they were missing for canvas and maps. Maybe @kobelb can further explain?

Reply:
It's being consumed by https://github.com/elastic/kibana/pull/34896/files#diff-1bb2390b8f55e83f3a830c385a7d664eR70 to determine whether the "go to app" should be visible.